test/dlc_tests/ec2/pytorch/training/test_pytorch_training.py

```diff
@@ -136,6 +136,9 @@ def test_pytorch_train_dgl_cpu(pytorch_training, ec2_connection, cpu_only, py3_o
 @pytest.mark.model("mnist")
 @pytest.mark.parametrize("ec2_instance_type", PT_EC2_GPU_INSTANCE_TYPE, indirect=True)
```

Comment:
All the pytest skips in this file are applicable for huggingface images as well?